payloads/v9/channel.ts

```diff
@@ -19,22 +19,13 @@ export interface APIPartialChannel {
 	 * The id of the channel
 	 */
 	id: Snowflake;
-	/**
-	 * The type of the channel
-	 *
-	 * See https://discord.com/developers/docs/resources/channel#channel-object-channel-types
-	 */
-	type: ChannelType;
```

Comment:
I wouldn't remove this from the partial channel, but rather make it a generic property you can pass in `interface APIPartialChannel<Type extends ChannelType> { type: Type; }`

Reply:
I realized that keeping the `type` here actually didn't affect any behavior, so I'll just add it back

Reply:
actually nevermind